Dockerfile.redhat

```diff
@@ -73,6 +73,14 @@ RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.n
             vim \
             xz && \
             yum clean all
+
+# Build and install pugixml
+RUN git clone -b v1.13 https://github.com/zeux/pugixml && \
```

Comment:
Do we want to add some cleanup at the end of this layer?

Reply:
this is a build image and the repo is very small